docs/Installation.md

```diff
@@ -544,7 +544,7 @@ you should be able to use the
 API on localhost port 5000 as below. The command requires that `curl` is installed.
 
 ```sh
-curl -s -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
+wget -q -O - --header "Content-Type: application/json" --post-data '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo

```

Comment:
Please give a strong argument to change from curl to wget at this late stage. I am against the change for v2019.1.